plugins/modules/database/misc/redis.py

```diff
@@ -277,7 +278,10 @@ def main():
         name = module.params['name']
 
         try:  # try to parse the value as if it were the memory size
-            value = str(human_to_bytes(module.params['value'].upper()))
+            if re.match('^\s*(\d*\.?\d*)\s*([A-Za-z]+)?\s*$', module.params['value'].upper()):
```

Comment:
```suggestion
            if re.match(r'^\s*(\d*\.?\d*)\s*([A-Za-z]+)?\s*$', module.params['value'].upper()):
```
The `r` is absolutely essential.

Reply:
will fix, obviously did something wrong when removing whitespace